File: eureka/android/library/src/main/java/com/lifeplus/Database/dbTables/MealsTable.java
package com.lifeplus.Database.dbTables;

public class MealsTable {
    public static final String TABLE_NAME = "meals";

    public static class Cols {
        public static final String ID = "Id";
        public static final String SESSION_ID = "session_id";
        public static final String TIME = "time";
        public static final String TYPE = "type";
        public static final String UPDATE_DATE = "update_date";
        public static final String UPLOAD_DATE = "upload_date";

        public static final String MEALTYPE = "mealtype";
        public static final String UTCYEAR = "utcyear";
        public static final String UTCMONTH = "utcmonth";
        public static final String UTCDAY = "utcday";
        public static final String UTCHOUR = "utchour";
        public static final String UTCMINUTE = "utcminute";
        public static final String UTCSECOND = "utcsecond";
        public static final String NOTTAKENMEAL = "notTakenMeal";
        public static final String DETAILS = "details";
    }
}

File: eureka/ios/nativeLib/nativeLib/Database/dbTables/MealsTable.swift
//
//  MealsTable.swift
//  LifePlus
//
//  Created by work on 10/09/20.
//

import Foundation

public class MealsTable {
  public static let TABLE_NAME: String = "meals"

    public class Cols {
      public static let SESSION_ID: String = "session_id"
      public static let TIME: String = "time"
      public static let TYPE: String = "type"
      public static let UPDATE_DATE:String = "update_date"
      public static let UPLOAD_DATE:String = "upload_date"

      public static let MEALTYPE:String = "mealtype"
      public static let UTCYEAR:String = "utcyear"
      public static let UTCMONTH:String = "utcmonth"
      public static let UTCDAY:String = "utcday"
      public static let UTCHOUR:String = "utchour"
      public static let UTCMINUTE:String = "utcminute"
      public static let UTCSECOND:String = "utcsecond"
    }
}
